aiohttp_cors/urldispatcher_router_adapter.py

```diff
@@ -35,13 +35,14 @@ def route_methods(self, route: web.Route):
 
     def add_options_method_handler(self, route: web.Route, handler):
         method = "OPTIONS"
+        name = None
```

Comment:
Thus you add anonymous (unnamed) route for OPTIONS method, right?
Please don't use `name=None` variable for this but pass `None` explicitly.